content/guides/guides/cross-origin-testing.md

```diff
@@ -129,6 +129,8 @@ specific test when it runs. For more information on this, please see our Web
 Security page regarding
 [Different superdomain per test requires cy.origin command](/guides/guides/web-security#Different-superdomain-per-test-requires-cy-origin-command).
 
+::include{file=partials/document-domain-workaround.md}
```

Comment:
Was this mean to be under Other Workarounds?

Reply:
I put it here because I feel like it is a work around, but it kind of feels thrown in there. Should we maybe dedicate a paragraph or something without the partial which makes more sense?

Reply:
Yeah I might add some context on why its there and indicate it's type of work-around

Reply:
What I did was just moved this up in cross origin testing in 3d4fb3e42 since we describe there why we do document.domain injection and it seems like the appropriate place to plug this. Not sure if this was there before but Revisiting I think it makes sense:
<img width="744" alt="Screenshot 2023-01-06 at 5 08 33 PM" src="https://user-images.githubusercontent.com/3980464/211108321-a5bb0af6-b43c-4564-aeea-01599e5cfc8d.png">